src/archiefvernietigingscomponent/templates/start_page.html

```diff
@@ -65,15 +65,52 @@ <h1 class="display-4">{{ settings.PROJECT_NAME }}</h1>
             will be reset multiple times per day.
           </p>
   
-          <p>
-            Here are the roles available in the demo:
-            <ul class="list-unstyled">
-              <li>Functional administrator</li>
-              <li>Process owner</li>
-              <li>Archivist</li>
-              <li>Record manager</li>
+          <p>Here are the roles available in the demo:</p>
+
+          <div class="container text-left">
+
+            <ul class="nav nav-tabs justify-content-center" id="roleChoiceEnglish" role="tablist">
+              <li class="nav-item ">
+                <a class="nav-link active text-info" id="record-manager-en-tab" data-toggle="tab" href="#record-manager-en" role="tab" aria-controls="record-manager-en" aria-selected="true">Record manager</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link text-info" id="process-owner-en-tab" data-toggle="tab" href="#process-owner-en" role="tab" aria-controls="process-owner-en" aria-selected="false">Process owner</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link text-info" id="archivist-en-tab" data-toggle="tab" href="#archivist-en" role="tab" aria-controls="archivist-en" aria-selected="false">Municipal archivist/supervisor</a>
+              </li>
             </ul>
-          </p>
+
+            <div class="tab-content" id="roles-en">
+              <div class="tab-pane fade show active" id="record-manager-en" role="tabpanel" aria-labelledby="record-manager-en-tab">
+                <br>
+                <ul>
+                  <li>The recordmanager initiates the proces to destroy cases and generates the list of destruction.</li>
+                  <li>He executes quality control over the list, checks if there are no relations that can’t be disconnected when the file is destroyed and checks if there are no cases on the list that can’t be destructed because it’s an exemption due to the criteria mentioned in the ‘selectielijst’ (this is the archiving policy list with all the storage periods per zaaktype/proces and allowed excemptions).</li>
+                  <li>Makes sure the list of destruction is forwarded to the procesowner.</li>
+                </ul>
+              </div>
+              <div class="tab-pane fade" id="process-owner-en" role="tabpanel" aria-labelledby="process-owner-en-tab">
+                <br>
+                <ul>
+                  <li>As the owner of the information he will do a check if all the cases on the list can be destroyed. Asks the question: Is the file still needed for the current business operation?</li>
+                  <li>Excempts files that are still needed and gives the argumentation for that.</li>
+                  <li>Approves the list, so it can be forwarded (in exctracted form/proces verbaal) tot the municipal archivist/supervisor. Or sents the list back tot he recordmanager, if there are exceptions/exemptions.</li>
```

Comment:
```suggestion
                  <li>Approves the list, so it can be forwarded (in exctracted form/proces verbaal) to the municipal archivist/supervisor. Or sends the list back to the record manager, if there are exceptions/exemptions.</li>
```